rudder-web/src/main/scala/bootstrap/liftweb/checks/CheckDSCSystemPolicy.scala

```diff
@@ -105,7 +105,7 @@ class CheckDSCSystemPolicy(
       , new Attribute("cn", "Rudder system policy: base configuration for DSC based agent")
       , new Attribute("description", "Rudder system policy: base configuration for DSC based agent")
       , new Attribute("longDescription", "This rule manage based configuration for DSC agent")
-      , new Attribute("ruleTarget", "group:dsc-based-agent")
+      , new Attribute("ruleTarget", "group:all-nodes-with-dsc-agent")
```

Comment:
ha - we should probably trigger a dynamic group refresh if this group is created